skyreader/plot/plot.py

```diff
@@ -836,7 +836,7 @@ def bounding_box(ra, dec, theta, phi):
                 )
         plt.legend(fontsize=6, loc="lower left")
         # Dump the whole contour
-        path = unique_id + ".contour.pkl"
+        path = self.output_dir / f"{unique_id}.contour.pkl"
         print("Saving contour to", path)
         with open(path, "wb") as f:
             pickle.dump(saving_contours, f)
```

Comment:
I recommend separating this into a separate helper function. The key indicator is that `path` is not used beyond this chunk. The same goes for the `ascii.write()` chunk above.

Reply:
Probably a good idea. I added a private method to save the contours now.